application/templates/cms/search_data_source.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% block pageTitle %}Search for an existing data source{% endblock %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for('static_site.topic', topic_slug=topic.slug), "text": topic.title}
+  ]
+%}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form method="GET" action="{{ url_for('cms.search_data_sources', topic_slug=topic.slug, subtopic_slug=subtopic.slug, measure_slug=measure.slug, version=measure_version.version )}}">
+
+        <div class="govuk-form-group">
+
+          <span class="govuk-caption-xl">{{ measure_version.title }} </span>
+          <h1 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--xl" for="q">Search for an existing data source</label>
+          </h1>
+
+          <input class="govuk-input govuk-input--width-30" id="q" name="q" type="text" value="{{ q }}">
+        </div>
+
+
+        <button class="govuk-button">Search</button>
+
+      </form>
+
+      {% if not data_sources == None %}
+
+        {% if data_sources|length > 0 %}
+
+          {% if data_sources|length == 100 %}
+
+            <p class="govuk-body">Only the first 100 results shown below.</p>
+          {% endif %}
+
+          <ul class="govuk-list">
+            {% for data_source in data_sources %}
+
+              <li>
+                <div class="govuk-heading-s govuk-!-margin-bottom-2">{{ data_source.title }}</div>
+                <div class="govuk-body">
+                  {% if data_source.publication_date %}
+                    <span class="eff-!-grey-1">Published on {{ data_source.publication_date }}</span>
+                  {% endif %}
+                  {% if data_source.source_url %}
+                    <a class="govuk-link" href="{{ data_source.source_url }}">View publication</a>
+                  {% endif %}
+                </div>
+              </li>
+
+            {% endfor %}
+          </ul>
+        {% else %}
+          <p class="govuk-body">No results found.</p>
```

Comment:
Can we include a recommended action for the user to take? This was common on previous search functionality I've worked on before.

e.g. "No results found. Please try again with different words or a less specific search."

Reply:
Good idea, have added (minus the "please"). We'll need to re-evaluate this when we add the "or add new data source" option.